packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -2965,6 +2965,64 @@ describeWithDOM('mount', () => {
 
       expect(mount(<Comp />).debug()).to.equal('<Comp />');
     });
+
+    describe('child components', () => {
+      class Child extends React.Component {
+        constructor(...args) {
+          super(...args);
+          this.state = { a: 'a' };
+        }
+
+        render() {
+          const { _ } = this.props;
```

Comment:
I find using `_` here makes this harder to read, it took me a while to realize this was getting passed through as `Child` props. I think I might be used to ignoring `_`, but maybe others might be the same 🤷‍♂️